crypto/evp/pmeth_lib.c

```diff
@@ -94,8 +94,15 @@ const EVP_PKEY_METHOD *EVP_PKEY_meth_find(int type)
 {
     pmeth_fn *ret;
     EVP_PKEY_METHOD tmp;
-    const EVP_PKEY_METHOD *t = &tmp, **ret;
-
+    const EVP_PKEY_METHOD *t = &tmp;
+
+    tmp.pkey_id = type;
+    if (app_pkey_methods) {
+        int idx;
+        idx = sk_EVP_PKEY_METHOD_find(app_pkey_methods, &tmp);
+        if (idx >= 0)
+            return sk_EVP_PKEY_METHOD_value(app_pkey_methods, idx);
+    }
```

Comment:
Make sure the documentation matches this change.

Reply:
That's a change from master that showed up here because of a wonky rebase.  It doesn't show up in the diff.